### DISCUSSION TITLE
红米 AX3000 (AX6) 解锁 SSH 教程

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/blog#12] [Comments]
https://blog.imouto.in/post/openwrt/2022/exploit-ssh-on-redmi-ax6/

**Comment by zhangjibiao** [comment]
感谢感谢，已成功解锁SSH！！！！！！

**Comment by cxwx** [comment]
猴米新 AX3000 产品 指的是什么？

**Comment by 1sm23** [comment]
一键换机过程中请求对端接口失败 code 1643

**Comment by 1715173329** [comment]
> 一键换机过程中请求对端接口失败 code 1643

无线没连上，多试几次，如果还不行就换一台用于操作的设备。

**Comment by 1sm23** [comment]
> 一键换机过程中请求对端接口失败 code 1643

出现这个错误信息和代码的朋友，第二步执行的时候也要重新从ax6后台获取新的TOKEN，亲测成功

**Comment by ghost** [comment]
请求YouTube出一个视频，新手看不懂

**Comment by WLaoDuo** [comment]
请问不支持的设备是指ra81吗？

**Comment by zhoudahong** [comment]
请教下红米AX3000有刷uboot和op教程吗，谢谢

**Comment by SakuraChanNya** [comment]
请教下，现在的AX6是哪个产品？听说现在的官网卖的AX3000不是本帖说的AX6

**Comment by 1715173329** [comment]
AX6 ra69 已经停产了，现在只有二手平台能买到。
现在的 ax3000 是 ra80 ra81 之类的，纯属垃圾。

**Comment by Tz-george** [comment]
那么问题来了，哪里来的openwrt的路由器A呢？

**Comment by 1sm23** [comment]
> 那么问题来了，哪里来的openwrt的路由器A呢？

我是买的二手香橙pi，需要可以卖给你😝

**Comment by KaranocaVe** [comment]
> 那么问题来了，哪里来的openwrt的路由器A呢？

虚拟机

**Comment by Tychristine** [comment]
# Redmi AX3000 ra81 已无法解锁SSH
**型号** ra81
**固件版本** miwifi_ra81_all_03430_1.0.33.bin

### 详细情况
按照本文的第3个步骤：令目标路由与OpenWRT连接并扩展WIFI时，请求返回的code:0
此时路由器的无线局域网密码没有变成SSH密码，其22端口也没有打开(复试后依然如此)

*估计只能走TTL途径了*

**Comment by 0xTimi2233** [comment]
固化完怎么恢复，想退货了
